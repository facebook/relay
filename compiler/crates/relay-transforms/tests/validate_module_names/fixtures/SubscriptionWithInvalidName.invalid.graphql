#expected-to-throw
subscription SubscriptionWithAnyOtherNameIsInvalidSubscription {
    feedbackLikeSubscribe(input: null) {
        clientMutationId
    }
}
