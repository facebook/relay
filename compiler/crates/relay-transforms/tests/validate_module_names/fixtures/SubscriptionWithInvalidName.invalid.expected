==================================== INPUT ====================================
#expected-to-throw
subscription SubscriptionWithAnyOtherNameIsInvalidSubscription {
    feedbackLikeSubscribe(input: null) {
        clientMutationId
    }
}
==================================== ERROR ====================================
✖︎ Subscriptions in graphql tags must start with the module name ('SubscriptionWithInvalidName') and end with 'Subscription'. Got 'SubscriptionWithAnyOtherNameIsInvalidSubscription' instead.

  SubscriptionWithInvalidName.invalid.graphql:2:14
    1 │ #expected-to-throw
    2 │ subscription SubscriptionWithAnyOtherNameIsInvalidSubscription {
      │              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    3 │     feedbackLikeSubscribe(input: null) {
